src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget/StakingWidgetProvider.tsx

```diff
@@ -0,0 +1,283 @@
+import React, {
```

Comment:
This file contains all of the various calculations depended on by the staking widget and its subcomponents